### PR TITLE
Shuts down the Summit's site, to prepare for next years.

### PR DESCRIPTION
**Pull request by AnUnsocialPigeon**
Adds a lil' button there instead of last years schedule. Took the link to the SU page of last year's summit to be the link of the current. Implemented a "notice" TSX component which can be used in replacement of a disabled button. The benefit of this is that it allows for inner html to be displayed within the confines of the button-like exterior. 

Furthermore, adds new line support to the button class.

Also added a konami-code easter egg ;) (https://simple.wikipedia.org/wiki/Konami_Code)

Added framework for a "Hall Of Fame" site path, ready as a further Easter egg to be an area where Dev's can each leave their notice on the site. This is currently under development, and will probably end up using the components that will be made for the "Who Are We" site which will also come soon!

Also added a TODO.md list to better handle TODO's

### CODE REVIEW

File: src/app/competition/page.tsx [hunk 1]
```diff
@@ -5,23 +5,24 @@ import defaultPage, { defaultPageWithScroll } from "@/components/default";
...
 
   if (!submissionOpen) {
     return defaultPage(
       <div>
-        <Button
-          enabled={false}
-          href=""
-          text="Submissions are not currently open."
-        />
+        <Notice>
```
Comment (JakubCzarlinski) [inline src/app/competition/page.tsx:16]: nice hahahah i forgot about this being a "button"

File: src/components/notice.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  const importantClassNames = props.important ? `bg-yellow-100 border-yellow-500 text-yellow-700` : ``;
+  
+  const allClassNames = `${baseClassNames} ${props.extraClasses || ""} ${importantClassNames || ""}`;
+
+  return (
+    <div className={allClassNames}>
+      {props.children}
+    </div>
+  );
+}
```
Comment (JakubCzarlinski) [inline src/components/notice.tsx:18]: pls configure your editor to terminate your files with `\n\n` ie the last line here would be `}\n\n`

File: src/components/button.tsx [hunk 3]
```diff
@@ -8,21 +8,29 @@ export default function Button(props: {
   ariaLabel?: string;
 }) {
   const classNames = `shadow-sm shadow-wai-gray bg-pure-white text-xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center`;
+  
+  // To allow for \n linebreaks
+  const formattedText = props.text.split('\n').map((line, index) => (
```
Comment (JakubCzarlinski) [inline src/components/button.tsx:13]: have you considered https://stackoverflow.com/questions/39325414/line-break-in-html-with-n

File: src/components/notice.tsx (hunk 2 above)
Comment (JakubCzarlinski) [reply src/components/notice.tsx:18]: github shows a big red stop sign that i dont like looking at when you go open the file and scroll down to the end of it



also https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

File: src/components/button.tsx (hunk 3 above)
Comment (AnUnsocialPigeon) [reply src/components/button.tsx:13]: i did, it didn't work, i did again, it work. FML.

File: src/components/button.tsx [hunk 4]
```diff
@@ -8,21 +8,25 @@ export default function Button(props: {
   ariaLabel?: string;
 }) {
   const classNames = `shadow-sm shadow-wai-gray bg-pure-white text-xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center`;
+  
+  // To allow for \n linebreaks
+  const textStyle = { whiteSpace: 'pre-line' };
```
Comment (JakubCzarlinski) [inline src/components/button.tsx:13]: we use tailwind for this project, so add `whitespace-pre-line` to the class instead of using `style`

Comment (AnUnsocialPigeon) [reply src/components/button.tsx:13]: Never did get my head round tailwind...

File: src/app/halloffame/page.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+// Summit details and ticket purchase page.
```
Comment (JakubCzarlinski) [inline src/app/halloffame/page.tsx:1]: rename dir to hall_of_fame

**Pull request merged**